parking_lot_src/ParkingLot/Bike.java

```diff
@@ -0,0 +1,9 @@
+package ParkingLot;
+
+public class Bike extends Vehicles{
+    public Bike(String color, String regNum) {
+        this.color = color;
```

Comment:
Make these variable private to achieve encapsulation, otherwise anyone from outside can change it. 
and add getter to fetch it

```
Example 
Bike bike = new Bike("red", "KA-1234");
bike.color = "blue";
` ``